src/member/models.py

```diff
@@ -117,6 +117,7 @@ class Meta:
         'member.Address',
         verbose_name=_('address'),
         null=True,
+        on_delete=models.CASCADE
```

Comment:
I think we might use models.SET_NULL. Deleting an address doesn't necessarily remove the member. How do you think?